index.css

```diff
@@ -1,3 +1,117 @@
 html {
-  
+  font-family: 'Indie Flower', cursive;
+}
+
+h1 {
+  font-size: 4rem;
+  text-align: center;
+  margin-bottom: 0;
+}
+
+.raw {
+  height: 35rem;
+  overflow: visible;
+}
+
+.container-one {
+  height: 34rem;
+  width: 22rem;
+  display: inline-block;
+  overflow: auto;
+}
+
+#one-player, #two-player, #reset {
+  height: 3rem;
+  width: 9rem;
+  font-size: 1.5rem;
+  font-family: 'Indie Flower', cursive;
+  /*border: solid magenta;*/
```

Comment:
It's always good code etiquette to remove any unnecessary/commented out code before submitting a final PR.